### ISSUE TITLE
USER ISSUE: Cannot sell into cart or wheelbarrow

### ISSUE THREAD

**Issue by kittle31** [StrangeLoopGames/EcoIssues#24474]
\
**User Id: (Not logged user)**
**Version:** 0.10.0.3 beta release-547

**Steps to Reproduce:**
1. goto a store you do not own
2. deploy a wood card (or small wood cart) or wheelbarrow right nexto you
3. attempt to buy large qty of something - ex: 300 raw Sausage, or 600 Huckleberries, or 200 Flour
4. ERR: not enough room

**Expected behavior:**
Same behavior as V9.  items that do not fit in player inventory are auto-placed into your wooden cart (or small wood cart)

**Actual behavior:**
ERR not enough room

**Do you have mods installed? Does the issue happen when no mods are installed?:**

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by D3nnis3n** [comment]
Is the cart linked to the store, otherwise this doesn't work?

**Comment by kittle31** [comment]
How do I link my own cart to a store owned by somebody else?   is this a new game mechanic for V10?

Let me try again.
As a cook, I frequently need a large quantity of items to cook - example: Huckleberries and Pumpkins.
So I find a store that sells Huckleberries and Pumpkins, and try to buy 1,000 of each.  This of course does not fit into the normal player inventory.  "error: not enough room"
in V9.x I could place a wooden cart or small wooden cart nexto the store and complete the purchase.   My player inventory would fill up with some of the items, and the remaining items would be placed in my cart.  I could complete the purchase in 1 transaction.

In V10 nothing is placed in the cart.  instead I receive an error "not enough room".  So instead I have to make many smaller purchases, and manually transfer each batch of the purchased items from my player inventory to my cart.  This is time frustrating, time consuming and IMO, places unneeded load on the server to handle all the transactions.

**Comment by kittle31** [comment]
minor updates to OP

**Comment by D3nnis3n** [comment]
When in the store interface there is a "storage" tab, make sure your cart is visible there and linked.

**Comment by D3nnis3n** [comment]
Closing due to not reproducable and no furhter information provided.
